qiskit/algorithms/quantum_time_evolution/variational/var_qrte.py

```diff
@@ -57,9 +59,11 @@ def __init__(
             error_based_ode: If False use the provided variational principle to get the parameter
                                 updates.
                              If True use the argument that minimizes the error error_bounds.
+                             Deprecated if error is not being computed
             ode_solver_callable: ODE solver callable that follows a SciPy OdeSolver interface.
             optimizer: Optimizer used in case error_based_ode is true.
             epsilon: # TODO, not sure where this will be used.
+            # TODO @DAL Where did you get this epsilon from? It's not in the original code.
```

Comment:
I think the plan might have been to parametrize checks for small numerical deviations from PSD or negligible imaginary parts. Might be an overkill.